CODE_OF_CONDUCT.md

```diff
@@ -2,50 +2,45 @@
 
 ## Our Pledge
 
-We as members, contributors, and leaders pledge to make participation in our
-community a harassment-free experience for everyone, regardless of age, body
-size, visible or invisible disability, ethnicity, sex characteristics, gender
-identity and expression, level of experience, education, socio-economic status,
-nationality, personal appearance, race, religion, or sexual identity
-and orientation.
+As members, contributors, and leaders, we pledge to ensure that everyone can participate 
+in our community without harassment regardless of age, body size, visible or invisible disability,
+ethnicity, sex characteristics, gender identity, or level of experience.
```

Comment:
You removed education, socio-economic status, nationality, race, religion and sexual orientation. Please add them back.